RevenueCatUI/Templates/V2/PaywallsV2View.swift

```diff
@@ -241,15 +241,15 @@ private struct LoadedPaywallsV2View: View {
             ComponentsView(
                 componentViewModels: [paywallState.componentViewModel],
                 onDismiss: self.onDismiss
-            )
+            )            
         }
         .environmentObject(self.selectedPackageContext)
         .frame(maxHeight: .infinity, alignment: .topLeading)
         .backgroundStyle(
             self.paywallState.componentsConfig.background
                 .asDisplayable(uiConfigProvider: uiConfigProvider).backgroundStyle
         )
-        .edgesIgnoringSafeArea(.top)
+        .edgesIgnoringSafeArea(.bottom)
```

Comment:
Yeah, this makes sense. Having `.edgesIgnoringSafeArea(.top)` could mean trouble when the paywall got presented in full cover